lib/iser.c

```diff
@@ -812,6 +816,39 @@ static int iser_create_iser_conn_res(struct iser_conn *iser_conn) {
 	return ret;
 }
 
+/*
+ * iser_rdma_set_option() - set rdma options
+ * currently support LIBISCSI_RDMA_ACK_TIMEOUT
+ * @cma_id:    connection manager id
+ * @iscsi:   iscsi context
+ */
+#ifdef HAVE_RDMA_ACK_TIMEOUT
+int iser_rdma_set_option(struct rdma_cm_id *cma_id, struct iscsi_context *iscsi) {
+	int ret = 0;
+	char *value;
+
+	value = getenv("LIBISCSI_RDMA_ACK_TIMEOUT");
```

Comment:
Different connections may require a different RDMA ack timeout. So I think what @sahlberg proposed is a better approach, namely to retrieve the ack timeout from the iSCSI URL.